File: Python/Account.py
class Account:
    def __init__(self, name, document, email, password):
        self.__name = name
        self.__document = document
        self.__email = email
        self.__password = password

    def __str__(self):
        return f"Nombre: {self.__name}\nDocumento: {self.__document}\nEmail: {self.__email}"

File: Php/Account.php
<?php 
class Account {
    private $id;
    private $name;
    private $document;
    private $email;
    private $password;

    public function __construct($name, $document, $email, $password){
        $this->name = $name;
        $this->document = $document;
        $this->email = $email;
        $this->password = $password;
    }

    public function __toString(){
        return "Name: ".$this->name."\nDocument: ".$this->document."\nEmail: ".$this->email;
    }
}
?>